plugins/afl-scoreboard/manager.py

```
@@ -0,0 +1,961 @@
+            except (TypeError, ValueError):
+                pass
+        return None
+
+    def _get_game_duration(self, mode_type: str, manager=None) -> float:
+        if manager is not None:
+            manager_duration = getattr(manager, "game_display_duration", None)
+            if manager_duration is not None:
+                return float(manager_duration)
+        return float(self.config.get(f"{mode_type}_game_duration", 15))
+
+    def get_cycle_duration(self, display_mode: str = None) -> Optional[float]:
```

Ruff (0.15.21)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)